docs/start_docs_server.sh

```diff
@@ -11,13 +36,8 @@ while true; do
   echo "Waiting for 15 seconds before starting to watch for file changes..."
   sleep 15
   
-  if [[ "$OSTYPE" == "darwin"* ]]; then
-    CHANGED_FILE=$(fswatch -1 --exclude 'docs/static/api-reference-docs' --exclude 'build/' --exclude '/docs/.docusaurus' --exclude 'changes.log' --exclude 'docs/node_modules/.cache/webpack' --exclude '.git/'  $WATCH_PATH)
-    echo "Detected changes in: $CHANGED_FILE"
-  else
-    echo "OS not supported"
-    exit 1
-  fi
+  CHANGED_FILE=$(fswatch -1 --exclude 'docs/static/api-reference-docs' --exclude 'build/' --exclude '/docs/.docusaurus' --exclude 'changes.log' --exclude 'docs/node_modules/.cache/webpack' --exclude '.git/'  $WATCH_PATH)
```

Comment:
```suggestion
  CHANGED_FILE=$(fswatch -1 --exclude 'docs/static/api-reference-docs' --exclude 'build/' --exclude '/docs/.docusaurus' --exclude 'docs/node_modules/.cache/webpack' --exclude '.git/'  $WATCH_PATH)
```

Reply:
the diff is weird but trying to not exclude changes.log as that file no longer is created by the script.